code/kobe.py

```diff
@@ -17,19 +17,23 @@
 import pprint
 import matplotlib.pyplot as plt
 import matplotlib as mpl
+import random
 
 import seaborn as sns
 sns.set()
 
 from scipy import constants as sp_constants
 from astropy import constants as astropy_constants
 from astropy import units as astropy_units
+from astropy import units as u
 
-
-# from mymodules import dataread, analysis, plottingroutine
+sys.path.insert(0, 'C:/Users/PR/Desktop/Masters_thesis')
```

Comment:
This line may be problematic.. Should we create a flag in kobe choice? It could go like: flag_working_pc - with 3 possible options: LM, PR, Horus. If this flag is set to 'PR', then we should allow this sys path line, otherwise another relevant code line.